src/sql/src/plan.rs

```diff
@@ -155,6 +155,10 @@ pub enum Plan {
         object_columns: usize,
         desc: RelationDesc,
     },
+    ReadRows {
```

Comment:
Super minor nit, but I think we should just call this `CopyFrom` (which mirrors e.g. `Insert`). The `SendRows` and `SendDiffs` variants are on their way out.